### PR TITLE
Update README - Added tip for UMASK

### PR DESCRIPTION
**Pull request by auanasgheps**
Since many users (including me) get confused about UMASK, I have added a little tip which I've found useful with other containers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ You can set many parameters when you configure this container, but you must spec
 | `MYJD_PASSWORD=foo` | Your MyJDownloader password |
 | `MYJD_DEVICE_NAME=goofy`| The device name that will appear on MyJdownloader portal |
 | `XDG_DOWNLOAD_DIR=/opt/JDownloader/Downloads` | If you use this variable, set it as per the downloads folder volume! |
-| `UMASK=0777` | Defines specific rights for your downloaded files (default: undefined) - Must respect octal form (begins with 0 followed by three numbers between 0 and 7 included) (cf. https://en.wikipedia.org/wiki/Umask) |
+| `UMASK=0777` | Defines specific rights for your downloaded files (default: undefined) - Must respect octal form (begins with 0 followed by three numbers between 0 and 7 included) (cf. https://en.wikipedia.org/wiki/Umask) - **TIP** If you want your downloads generally available in your system, you can set a value of `002` which assigns full control to owner and group, whilst others can read and execute.|
```
Comment (jaymoulin) [inline README.md:72]:
```suggestion
| `UMASK=0777` | Defines specific rights for your downloaded files (default: undefined) - Must respect octal form (begins with 0 followed by three numbers between 0 and 7 included) (cf. https://en.wikipedia.org/wiki/Umask) - **TIP** If you want your downloads generally available in your system, you can set a value of `0775` which assigns full control to owner and group, whilst others can read and execute.|
```

Comment (jaymoulin) [reply README.md:72]: Also, it's better for neophytes not defining UMASK environment variable

Comment (auanasgheps) [reply README.md:72]: I may not have enough expertise, but as far as I know UMASK does not follow the octal "classic" permissions.
Moreover If you take a look at [linuxserver containers](https://github.com/linuxserver/docker-qbittorrent#docker) they use the format I proposed; I follow their templates/style :)

Comment (jaymoulin) [reply README.md:72]: My bad on 0775. Please don't copy linuxserver container's work, they do what they do. Umask command can understand both octal and decimal forms (as it interpret three digits numbers as octal). I don't want to debug eventual errors on umask so I rather force people to use octal form as they should with chmod

Comment (jaymoulin) [reply README.md:72]: For information purpose, 0002 umask result to a 0775 permission mod

Comment (auanasgheps) [reply README.md:72]: I didn't know about Octal support by UMASK, nice to know.
Well, if you still like this PR just change the UMASK value to whatever fits you.

Comment (jaymoulin) [reply README.md:72]: I don't actually really think putting umask 0002 will resolve anything as rights in a container completely depends on the user executing the program, therefore, relation with the `-u` parameter. Putting umask 0002 may result in a security breach by authorizing any user to execute a downloaded program. A user executing the container without `-u` parameter will be able to read and execute downloaded programs with UMASK 0002 but not delete as any other user. Executing the container with a correct -u parameter will change downloaded files owner and group resulting in a different umask interpretation

**Pull request closed without merging**